scripts/us_census/pep/population_estimate_by_race/preprocess.py

```diff
@@ -724,14 +900,34 @@ def _load_data(self, file: str) -> pd.DataFrame:
                     df['geo_ID'].map(str)).str[:len('geoId/NN')]
                 df_state = df_state.groupby(['Year', 'geo_ID']).sum().\
                     reset_index()
-                df = df.append(df_state, ignore_index=True)
-                df = df.append(df_national, ignore_index=True)
+                df = pd.concat([df, df_state], ignore_index=True)
+                df = pd.concat([df, df_national], ignore_index=True)
```

Comment:
concat in a single statement is faster:
`df = pd.concat([df, df_state, df_national], ignore_index=True)`

Reply:
modified as suggested